packages/appkit/src/client/appkit-base-client.ts

```diff
@@ -469,18 +472,67 @@ export abstract class AppKitBaseClient {
         }
       },
       disconnect: async (chainNamespace?: ChainNamespace) => {
-        const namespace = chainNamespace || (ChainController.state.activeChain as ChainNamespace)
-        const adapter = this.getAdapter(namespace)
-        const provider = ProviderUtil.getProvider(namespace)
-        const providerType = ProviderUtil.getProviderId(namespace)
+        const chainsToDisconnect = getChainsToDisconnect(chainNamespace)
+
+        try {
+          // Reset send state when disconnecting
+          SendController.resetSend()
```

Comment:
If we consider code splitting / lazy loading in mind, should we dynamically import SendController (and other feature controllers) here?

Reply:
maybe? honestly this was already imported in the other controller so it doesnt matter. 
I'd rather split in a separate pr than omplicate this one further